### PR TITLE
clean up img/decoding

### LINKED ISSUE #1388: "Content attributes:" of <img> element
https://w3c.github.io/html/semantics-embedded-content.html#the-img-element
> Content attributes:
> - Global attributes
> - alt - Replacement text for use when images are not available
> - src - Address of the resource
> - srcset - Images to use in different situations (e.g., high-resolution displays, small monitors, etc)
> - sizes - Image sizes between breakpoints
> - crossorigin - How the element handles crossorigin requests
> - usemap - Name of image map to use
> - ismap - Whether the image is a server-side image map
> - width - Horizontal dimension
> - height - Vertical dimension
> - referrerpolicy - Referrer policy for fetches initiated by the element
> - longdesc - A url that provides a link to an expanded description of the image, defined in  [html-longdesc]

→ `decoding` attribute should be added.

Ref: https://w3c.github.io/html/semantics-embedded-content.html#element-attrdef-img-decoding

### PR DESCRIPTION
**Pull request by chaals**
fix #1388
* Note it is at risk (but it is likely to be OK)
* Add it to `img` attributes list
* Add it to attributes index

### CODE REVIEW

File: sections/semantics-embedded-content.include [hunk 1]
```diff
@@ -898,13 +899,13 @@
   settings attribute</a>. Its purpose is to allow images from third-party sites that allow
   cross-origin access to be used with <{canvas}>.
 
-  The <dfn element-attr for="img"><code>referrerpolicy</code></dfn> attribute is a <a>referrer policy attribute</a>. Its purpose is to
-  set the <a>referrer policy</a> used when <a>fetching</a> the image. [[!REFERRERPOLICY]]
+  The <dfn element-attr for="img"><code>decoding</code></dfn> attribute is an enumerated attribute.
+  It is an  [=image decoding hint=], to reqeust synchronous asynchronous image loading.
```
Comment (prlbr) [inline sections/semantics-embedded-content.include]: I think there’s an "or" missing between "synchronous asynchronous".
(And does the comma have a function in the same sentence?)

Comment (chaals) [reply sections/semantics-embedded-content.include]: "or": yep. +typo in "request", and the comma is ok but unnecessary.

all fixed - thank you

File: sections/semantics-embedded-content.include [hunk 2]
```diff
@@ -898,13 +899,13 @@
   settings attribute</a>. Its purpose is to allow images from third-party sites that allow
   cross-origin access to be used with <{canvas}>.
 
-  The <dfn element-attr for="img"><code>referrerpolicy</code></dfn> attribute is a <a>referrer policy attribute</a>. Its purpose is to
-  set the <a>referrer policy</a> used when <a>fetching</a> the image. [[!REFERRERPOLICY]]
+  The <dfn element-attr for="img"><code>decoding</code></dfn> attribute is an enumerated attribute.
+  It is an [=image decoding hint=] to request synchronous or asynchronous image loading.
+  Valid values are "<code>async</code>", "<code>async</code>", and "<code>auto</code>".
```
Comment (mtrootyy) [inline sections/semantics-embedded-content.include:904]: '`are "<code>async</code>", "<code>async</code>", and`' is typo.

Comment (siusin) [reply sections/semantics-embedded-content.include:904]: Oops, sorry, didn't refresh the page before merging... will submit another commit to fix this.

Thanks for the review @mtrootyy .

**Comment by cynthia** [comment]
@chaals Why is this at risk? Safari and Chrome implement it.

https://bugs.webkit.org/show_bug.cgi?id=176016

**Comment by chaals** [comment]
@cynthia 

> Why is this at risk

To be pedantic, it's **potentially** at risk. Because i was being super-cautious - and as noted, I figured it was likely OK. 

In painful detail, the tests I was looking at only actually covered the IDL attribute, not useful behaviour, and the Chromium info I dug up to start suggested implementation was still in progress.

**Pull request merged**